vulnerabilities/data_source.py

```diff
@@ -43,6 +43,17 @@
 from vulnerabilities.oval_parser import OvalParser
 
 
+@dataclasses.dataclass
+class VulnerabilityReferenceUnit:
```

Comment:
Why not call it what it is? Simply ither a `VulnerabilityReference` or a `Reference`

Reply:
Calling it Reference